sdk/textanalytics/Azure.AI.TextAnalytics/src/GeneratorStubs/InternalTypes.cs

```diff
@@ -7,29 +7,89 @@ namespace Azure.AI.TextAnalytics.Models
 {
 #pragma warning disable SA1402 // File may only contain a single type
 
+    [CodeGenModel("AbstractiveSummarizationResultBaseDocumentsItem")]
```

Comment:
Something is very wrong here. They should all be `internal` by default. This becomes an expensive endeavor to update this every time. Did you try using a transform to set `x-accessibility: internal`?

Reply:
There are two categories of models that ended up here:

Category 1. Extensible enums. I was able to add a transform to the autorest.md to take care of these.

Category 2. Models that are defined in the swagger spec as the "items" in an array while also using the "allOf" property. For example, the swagger definition that generated the "ExtractiveSummarizationResultDocumentsItem" class: [[link]](https://github.com/Azure/azure-rest-api-specs/blob/6bb39b483e16fe68c1fda9aa55801d5f814aa327/dev/cognitiveservices/data-plane/Language/analyzetext.json#L2122). I found this issue on GitHub describing a similar problem: https://github.com/Azure/autorest.csharp/issues/1163. Then I noticed that other models in the same circumstances (such as the "CustomEntitiesResultDocumentsItem" class) were already here, so assumed this was the acceptable workaround for those.